src/node/node-nn-init-full.md

```diff
@@ -0,0 +1,204 @@
+
+- \\( \HYB \\) for the \\( \PtoP \\)-\\( \WS \\) Hybrid Network, for nodes operating
+in a unified network layer.
+
+The network layer manages:
+
+- \\( \Peer \\) discovery using _phonebook addresses_,
+
+- Connection pools, and
+
+- Message routing.
+
+> For further details on the network layer, refer to Algorand Network [non-normative specification](../network/network-overview.md).
+
+### Cryptography Resource Pools
+
+The node initializes worker pools for handling cryptographic tasks with priority
+queues:
+
+- \\( \CryptoPool \\) handles general-purpose cryptographic operations,
+
+- \\( \CryptoPool.\mathrm{lowPriority} \\) and \\( \CryptoPool.\mathrm{highPriority} \\)
+handle transaction verification, grouped by priority.
+
+> For further details on the transaction validation, see the Ledger [specification](../ledger/ledger-overview.md).
+
+> For further details on the cryptographic primitives and algorithms, see the Crypto [specification](../crypto/crypto-overview.md).
+
+### Ledger
+
+The node loads its local view of the blockchain state from disk (accounts, blocks, protocol data, etc.),
+based on the specified genesis configuration.
+
+If the Ledger is empty, it initializes the required structures.
+
+It also validates:
+
+- The genesis configuration, and
+
+- Ledger integrity.
+
+> For further details on the Ledger entities, see the Ledger [specification](../ledger/ledger-overview.md).
+
+### Account Management (Agreement)
+
+The node prepares for consensus participation and registered account tracking by:
+
+- Creating and managing a registry of participation keys,
+
+- Loading any pre-existing participation keys from disk,
+
+- Setting up participation key rotation.
+
+> For further details on the Algorand keys, see the Keys [specification](../keys/keys-overview.md).
+
+> For further details on the key registration transactions, see the Ledger [specification](../ledger/ledger-overview.md).
+
+### Transaction Pool
+
+The node creates the _Transaction Pool_ (\\( \TP \\)), which:
+
+- Accepts and validates new transactions,
+
+- Maintains the queue of uncommitted transactions,
+
+- Manages transaction synchronization across the network,
+
+- Creates Block Listeners reacting to new blocks to prune already committed transactions
+and update pending transactions.
+
+> For further details on the Transaction Pool, see the Ledger [non-normative specification](../ledger/ledger-nn-txpool.md).
+
+### Services
+
+Finally, the node launches all essential background services:
+
+<!-- TODO: Fix links once all chapters are finalized -->
+
+- [Catchup](#node-catchup) Service,
+
+- [Agreement](abft.md) Service,
+
+- [Transaction Pool](ledger-overview.md#transaction-pool) Syncer Service,
+
+- [Block](ledger.md#blocks) Service,
+
+- [Ledger](ledger.md) Service,
+
+- [Transaction](ledger.md#transactions) Handler,
+
+- [State Proof](crypto.md#state-proofs) Worker.
+
+- [Heartbeat](ledger.md#heartbeat-transaction) Service.
```

Comment:
I noticed these links aren't working. Are we going to update them all after the rest of the docs are included? Or shall I include suggestion/changes to them as I come across them?

Reply:
Please disregard broken links right now. There are still a lot of moving parts involved. We will have a final fix PR for all the links (and maybe a `pre-commit` or `CI` task to automate broken link detection, both internal and external).